packages/flutter/test/painting/image_cache_resize_test.dart

```diff
@@ -9,36 +9,74 @@ import '../rendering/rendering_tester.dart';
 import 'mocks_for_image_cache.dart';
 
 void main() {
-  new TestRenderingFlutterBinding(); // initializes the imageCache
+  group(ImageCache, () {
+     setUpAll(()  {
+       new TestRenderingFlutterBinding(); // initializes the imageCache
+     });
```

Comment:
What is the benefit of putting this in a `setUpAll` block instead of just in `void main` directly?

Reply:
None really, removed